### PR TITLE
Fix some format errors

### PR DESCRIPTION
**Pull request by miaoyq**
Signed-off-by: Yanqiang Miao <miao.yanqiang@zte.com.cn>

### CODE REVIEW

File: docs/reference/run.md [hunk 1]
```diff
@@ -345,7 +345,7 @@ Supported networks :
       <td class="no-wrap"><strong>container</strong>:&lt;name|id&gt;</td>
       <td>
         Use the network stack of another container, specified via
-        its *name* or *id*.
```
Comment (AkihiroSuda) [inline docs/reference/run.md]: img_20161129_195516

Please fix this format as well?

Comment (vdemeester) [inline docs/reference/run.md]: @AkihiroSuda the `{% raw %}` and `{% endraw %}` are needed though :angel:

Comment (thaJeztah) [inline docs/reference/run.md]: Ah, yes, that's due to the switch to Jekyll 😞 , which doesn't support Markdown in HTML tables; I think this is being addressed in https://github.com/docker/docker/pull/26791

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Review by vdemeester — APPROVED** [review]
LGTM 🐸

**Pull request merged**